packages/fuel-gauge/src/predicate.test.ts

```diff
@@ -1,11 +1,13 @@
 import { generateTestWallet } from '@fuel-ts/wallet/test-utils';
 import { readFileSync } from 'fs';
-import { Address, NativeAssetId, bn, toHex, toNumber, Provider, Predicate } from 'fuels';
+import { Address, NativeAssetId, bn, toHex, toNumber, Provider, Predicate, Interface } from 'fuels';
```

Comment:
`Interface` isn't used and can be removed.

```suggestion
import { Address, NativeAssetId, bn, toHex, toNumber, Provider, Predicate } from 'fuels';
```